packages/components/field-label/README.md

```diff
@@ -55,7 +55,8 @@ The `hintIcon` also accepts a custom `color` while defaulting to `warning` in th
 | `hintIcon`             | `node`             |    -     | -                            | -       | Icon to be displayed beside the hint text. Will only get rendered when `hint` is passed as well.                                                                                                                                                                      |
 | `badge`                | `node`             |    -     | -                            | -       | Badge to be displayed beside the label. Might be used to display additional information about the content of the field (E.g verified email)                                                                                                                           |
 | `hasRequiredIndicator` | `bool`             |    -     | -                            | `false` | Indicates if the labeled field is required in a form                                                                                                                                                                                                                  |
-| `htmlFor`              | `string`           |    -     | -                            | -       | ID of the labeled input                                                                                                                                                                                                                                               |
+| `htmlFor`              | `string`           |    -     | -                            | -       | ID of the labeled input, used for the `for` attribute                                                                                                                                                                                                                 |
+| `id`                   | `string`           |    -     | -                            | -       | ID of the labeled input                                                                                                                                                                                                                                               |
```

Comment:
This is the ID of the label not the input. Right?

Reply:
yes for the label. Should we name it differently?

Reply:
I am mentioning because the `description` indicates that it is the ID of the input.

Reply:
> Should we name it differently?

Name is correct just the description was missleading. May be updating the description would be enough